src/sentry/integrations/vsts/integration.py

```diff
@@ -453,7 +453,19 @@ def dispatch(self, request, pipeline):
         access_token = state['data']['access_token']
         user = get_user_info(access_token)
 
-        accounts = self.get_accounts(access_token, user['uuid'])['value']
+        accounts = self.get_accounts(access_token, user['uuid'])
+        self.logger(
+            'vsts.get_accounts',
+            extra={
+                # TODO(lb): I feel like there was a get org
+                # from the request user but I can't remember at all where it is
+                # 'organization_id': organization.id,
```

Comment:
I feel certain there's somewhere that had a function that got the org from the request.... does anyone remember what it's called?

Reply:
i don't think `PipelineView` inherits from `OrganizationEndpoint`

Reply:
Yea 😒

Reply:
soo i think the only way to get it would be from the pipeline state, which i don't totally know off the top of my head

Reply:
actually @lauryndbrown it looks like you can get it from `pipeline.organization` i think, but can you verify?

Reply:
@macqueen you're completely right. :)